File: Python/selecao/1041.py
"""
Problema: 1041 - Coordenadas de um ponto
"""
#lendo o ponto
x, y = input().split(" ")
#convertendo valores
x = float(x)
y = float(y)
#definido a resposta
if x > 0:
    if y > 0:
        print("Q1")
    elif y < 0:
        print("Q4")
    else:
        print("Eixo X")
elif x < 0:
    if y > 0:
        print("Q2")
    elif y < 0:
        print("Q3")
    else:
        print("Eixo X")
elif x == 0:
    if (y > 0) or (y < 0):
        print("Eixo Y")
    else:
        print("Origem")

File: C/Selecao/1041.c
/**
 * @file 1041.c
 * @author michaelthierry86@gmail.com
 * @brief Coordenadas de um ponto: Esse programa lê dois valores que representam
 * as coordenadas de um ponto. Em seguida informa a qual quadrante, eixo ou se o 
 * ponto esta na origem.
 * @version 0.1
 * @date 2022-04-18
 * 
 * @copyright Copyright (c) 2022
 * 
 */
/*Incluindo Biblioteca*/
#include <stdio.h>
/*Função principal*/
int main(){
    //Declarando variaveis
    float x, y;
    //Lendo do usuario
    scanf("%f %f", &x, &y);
    //Avaliando a situação
    if(x > 0 && y > 0){
        printf("Q1\n");

    }else if(x > 0 && y < 0){
        printf("Q4\n");

    }else if(x < 0 && y > 0){
        printf("Q2\n");

    }else if(x < 0 && y < 0){
        printf("Q3\n");

    }else if((x == 0) && (y > 0 || y < 0)){
        printf("Eixo Y\n");

    }else if((y == 0) && (x > 0 || x < 0)){
        printf("Eixo X\n");

    }else if((y == 0) && (x == 0)){
        printf("Origem\n");

    }
    return 0;
}